NuRadioReco/modules/efieldToVoltageConverter.py

```diff
@@ -126,6 +124,7 @@ def run(self, evt, station, det, channel_ids=None):
             channel_ids = det.get_channel_ids(sim_station_id)
 
         for channel_id in channel_ids:
+
```

Comment:
I think you and I perceive indentations differently :upside_down_face: For me the indentation is sufficient to indicate a break, so I don't see the need for an empty newline here. But this is just a comment, not a request for changes.

Reply:
Just a comment so that you maybe understand me better - often I am not refactoring code because I feel make this my explicit target. I read code in order to understand it and in order to understand it it is enormously helpful to me to add empty lines in code. Its like reading a book which does not has any paragraphs but just one big block of text is really tiring to read. But with code I need more often a break to appreciate the different steps.

Reply:
Yes, and in general I agree that your changes help to make the code more readable! I would just never add an empty line directly beneath a `for` loop for the same reason that I wouldn't do it underneath `def`, it is clear to me from the indentation that a new logical block starts.